### DISCUSSION TITLE
Gkape editor v1.3.0.2

### DISCUSSION THREAD

**Discussion by BouraouiMowamedWadii** [EricZimmerman/KapeFiles#934] [Q&A]
Hi,

I encountered a small problem with the Gkape editor.

In fact, I can't scroll down the table for targets and modules.

I am using version v1.3.0.2.

<img width="960" alt="gkape v1 3 0 2" src="https://github.com/EricZimmerman/KapeFiles/assets/24892944/47719ecd-5d79-42ed-90b2-fef45f3ada93">

**Comment by EricZimmerman** [comment]
It's your resolution. Not much I can do.

**Reply by BouraouiMowamedWadii** [reply]
Thank you Eric for your reply ^^

**Reply by AndrewRathbun** [reply]
> Thank you Eric for your reply ^^ 

Try changing the scaling in Windows and reopen gKAPE. You may have to adjust a few times to find a working solution.

**Reply by BouraouiMowamedWadii** [reply]
Yes it works well once changing the resolution.
thank you
